js/index.js

```diff
@@ -0,0 +1,235 @@
+var getWork = document.getElementById("new-todo-title"); // 할 일을 적는 input 태그
+var todoList = document.getElementById("todo-list"); // 작성한 할 일이 삽입되는 ul 태그
+var viewAllList = document.querySelector(".all"); // 전체 보기 버튼
+var viewTodoList = document.querySelector(".active"); // 해야할 일 보기 버튼
+var viewCompleteList = document.querySelector(".completed"); // 완료한 일 보기 버튼
+ 
+function init() {   // 페이지 로드 시 이벤트 리스너 부착
+  getWork.addEventListener("keypress", AddNewList);
+
+  viewAllList.addEventListener("click", viewAll);
+  viewTodoList.addEventListener("click", viewTodo);
+  viewCompleteList.addEventListener("click", viewDone);
+
+  window.addEventListener("beforeunload", saveLocalStorage);
+  window.addEventListener("DOMContentLoaded", loadLocalStorage);
+}
+
+function AddNewList(e) {  // 새로운 항목을 추가하는 기능 
```

Comment:
```suggestion
function AddNewList( { target, key }) {  // 새로운 항목을 추가하는 기능 
```
구조분해할당을 통해 인자를 전달받으면 코드가 더 깔끔해질거 같습니다!

Reply:
생소한 개념이라 아직 어떻게 사용해야할지 감이 잘 오지 않습니다. 추가적으로 공부하고 다음 코드에는 활용할 수 있도록 해보겠습니다!